demo/config.yaml

```diff
@@ -79,7 +77,7 @@ lcr-modules:
     utils:
         inputs:
             bed: 
-                grch37: "__UPDATE__" # make sure this corresponds with config["lcr-modules"]["picard_qc"]["inputs"]["intervals"]
+                grch37: "/projects/dscott_prj/CCSRI_1500/exomes/ref/agilent/hg19/target_regions.nochr.bed" # make sure this corresponds with config["lcr-modules"]["picard_qc"]["inputs"]["intervals"]
```

Comment:
I was using `__UPDATE__` here so that if tested outside GSC and not updated to the user-specific file, the oncopipe will exit pointing out to this. Otherwise, it will be showing error of missing file, and it might give hard time outside users to see where this missing file is coming from. Do you think it should be reverted to `__UPDATE__` when testing is done? Otherwise, I think it might be helpful to add another comment here for outside users.

Reply:
OK I think the actual solution here is to include the bed file in the repo. Otherwise the demo isn't really runnable, and there's nothing in the readme to help external users find this file. I'll put a copy there before I merge this. Thanks for the review!